bindings/pydeck/Makefile

```diff
@@ -31,7 +32,7 @@ setup-env:
 	test `command -v python3` || echo Please install python3
 	test `command -v yarn` || echo Please install yarn
 	test `command -v python3` && test `command -v yarn`
-	python3 -m venv env
+	python3 -m venv $(WORKON_HOME)
```

Comment:
This change was made to allow local installations to put the venv outside of the project directory. This is in order to avoid the following error when running make prepare-jupyter:

```
ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '/Volumes/CaseSensitive/deck.gl/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
    at file:///Volumes/CaseSensitive/deck.gl/bindings/pydeck/env/lib/python3.8/site-packages/jupyterlab/node-version-check.js:6:13
    at ModuleJob.run (node:internal/modules/esm/module_job:195:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:336:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:106:12)
```

I do not understand why this occurs only on local and not in the github actions test run.